tests/foreman/ui/test_puppet_classes.py

```diff
@@ -25,26 +23,23 @@ def test_update_positive(self, description):
         class_name = 'foreman_scap_client'
         param_name = 'ca file'
         with Session(self.browser):
-            # Importing puppet classes from puppet-foreman_scap_client
-            # module for update process
-            if self.puppetclasses.search(class_name) is None:
-                self.puppetclasses.import_scap_client_puppet_classes()
-            self.assertIsNotNone(self.puppetclasses.search(class_name))
-            self.puppetclasses.update_class_parameter_description(
-                class_name,
-                param_name,
-                description
-            )
-            self.assertEqual(
-                description,
-                self.puppetclasses.fetch_class_parameter_description(
-                    class_name,
-                    param_name)
-            )
-
-    @data(*generate_strings_list(len1=8))
+            for description in generate_strings_list(len1=8):
```

Comment:
I believe we can increase the length of description here. Usually we will have more than 8 chars in description. Also we can test how the description behaves with long strings.